### DISCUSSION TITLE
LoRA training generating blanks / black images

### DISCUSSION THREAD

**Discussion by gohan2091** [bmaltais/kohya_ss#1612] [Q&A]
When I use Network Rank 8 with Network Alpha as 1, my LoRA works fine but when I change these two settings to say 64/64, 128/64 or 64/32, I generate pure black images no matter what epoch version I use and lowering the weight doesn't help. Any ideas? I am using SD.Next (Vladamatic's fork of A1111). 

Just for comparison, this LoRA works: https://pastebin.com/ZF4F14Mz

but this one only shows blank images: https://pastebin.com/UMxLSJbg

A friend tried my LoRA in his A1111 and he gets this: https://i.ibb.co/r7PYB50/380600205-855779552922253-2284899826789538050-n.png

I tried again today with different images and again all black images no matter what epoch I use and no matter the weight. Here were my settings for today: https://pastebin.com/4HdmGBXD

**Comment by BrentWilkins** [comment]
I'm as lost as you might be, but when I've gotten black samples, I slow down the learning rate. I think this has always fixed it for me.
